packages/forms/src/widgets/templates/FieldTemplate/FieldTemplate.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import FieldTemplate from './FieldTemplate.component';
+
+describe('FieldTemplate', () => {
+	it('should render the common widget markup', () => {
+		// given
+		const props = {
+			descriptionId: 'test-description',
+			errorId: 'test-error',
+			label: 'Test',
+			description: 'This is the description',
+			error: 'This is the error',
+		};
+		// when
+		const wrapper = mount(
+			<FieldTemplate {...props}>
+				<input name="test" />
+			</FieldTemplate>,
+		);
+		// then
+		expect(wrapper.html()).toMatchSnapshot();
```

Comment:
Snapshotting a `.html()` make it difficult to read.
Can you use `toJson()` instead ?